sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/quickpulse/model/QuickPulseDependencyDocument.java

```diff
@@ -94,4 +82,55 @@ public String getOperationName() {
     public void setOperationName(String operationName) {
         this.operationName = operationName;
     }
+
+    @Override
+    public JsonWriter toJson(JsonWriter jsonWriter) throws IOException {
```

Comment:
in the relative near future, we will start using swagger to auto-generate quickpulse models.